bedrock/firefox/tests/test_views.py

```diff
@@ -231,7 +231,7 @@ def test_returns_valid_data(self):
         self.assertDictEqual(attrs, final_params)
         self.assertEqual(
             data["attribution_sig"],
-            "960fd9eabdf2890320457f8638062e8918812f896128afce5d50d2ab097270ad",
+            "f510544be8630ce0b73646a973fa011dc860ace01c3fdb55140d1e7c01c2df60",
```

Comment:
Presumably because a contributing URL has changed?

Reply:
That's right 👍